src/js/common/schemaform/reducers/index.js

```diff
@@ -21,12 +21,19 @@ function recalculateSchemaAndData(initialState) {
     .reduce((state, pageKey) => {
       const page = state[pageKey];
       // on each data change, we need to do the following steps
+
+      // Flatten the data from all the pages
+      const formData = _.mapValues('data', state);
```

Comment:
So, I think based on this let's just pass one variable, formData, that's a flattened version of the form. We should also update `updateSchema` to be consistent, but that should probably be a separate item because we need to update some existing configs.